Detectors/MUON/MID/Raw/src/CrateMasks.cxx

```diff
@@ -94,7 +94,7 @@ bool CrateMasks::load(const char* filename)
     std::getline(ss, token, ' ');
     uint16_t feeId = std::atoi(token.c_str());
     std::getline(ss, token, ' ');
-    uint8_t mask = std::atoi(token.c_str());
+    uint8_t mask = std::strtol(token.c_str(), nullptr, 16);
```

Comment:
don't you get a warning here ? strtol returns a long int that you're trying to fit into an unsigned char after all ...

Reply:
I rechecked, but I do not have warnings, at least on Mac OS.
I'll do a static cast anyways to be sure...